<launch>

  <arg name="name" default="uav1"/>

  <!-- Settings for ATAK communications -->
  <arg name="callsign" default="$(arg name)" doc="The callsign used by the robot on the ATAK system"/>
  <arg name="team_name" default="Ground Robots" doc="The Team Name used by the robot on the ATAK system"/>
  <arg name="team_role" default="Autonomous Agent" doc="The Team Role used by the robot on the ATAK system"/>
  <arg name="tak_ip" default="10.13.0.10" doc="The IP address of the ATAK server"/>
  <arg name="tak_port" default="8088" doc="The Port number on the ATAK server to connect to"/>
  <arg name="odom_topic" default="/uav1/mavros/global_position/local" doc="The odometry topic that declares the robot location"/>
  <arg name="baselink_frame" default="/uav1/base_link" doc="The base link frame of the robot"/>
  <arg name="global_frame" default="utm" doc="The global coordinate frame used to send global locations to ATAK"/>

  <group ns="$(arg name)/atak">
    <node name="tak_echo" pkg="atak_bridge" type="takmsg_echo.py" output="screen">
<!--    <node name="dcist_cots" pkg="atak_bridge" type="obj_2_atak_uav.py" output="screen">-->
        <param name="name" value="$(arg name)" />
        <param name="callsign" value="$(arg callsign)" />
        <param name="team_name" value="$(arg team_name)" />
        <param name="team_role" value="$(arg team_role)" />
        <param name="tak_ip" value="$(arg tak_ip)" />
        <param name="tak_port" value="$(arg tak_port)" />
        <param name="odom_topic" value="$(arg odom_topic)" />
        <param name="baselink_frame" value="$(arg baselink_frame)" />
        <param name="global_frame" value="$(arg global_frame)" />
    </node>
  </group> 



</launch>
